### DISCUSSION TITLE
How much will approximately cost for running this Project

### DISCUSSION THREAD

**Discussion by bganeshmail** [aws-samples/aws-saas-factory-eks-reference-architecture#55] [Q&A]
Dear Team,

This is excellent project and nicely built.

Please advice if we can run this test using AMAZON Free tier account and how much it cost if we run this code without any changes. Would like to run and see the result.

Thanks much,
Ganesh B

**Comment by oliwerhelsen** [comment]
Hi,

I think that is a hard question to answer. Not all the services provide a free tier and the cost depends on your project. Are using a custom domain? How many accounts for cod commit? How many clusters for EKS etc?

You can follow the billing under your user account and you can also set up alerts when reaching your budget. 

Hope that this helps you on the way forward.

Best regards Oliwer
